### PR TITLE
fix: missing backticks in code block and indentation in `add-content-collection`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Added missing backticks at the end of a code block and fixed identation.

#### Related issues & labels (optional)

- Suggested label: tutorial

### CODE REVIEW

File: src/content/docs/en/tutorials/add-content-collections.mdx [hunk 1]
```diff
@@ -248,6 +248,7 @@ The steps below show you how to extend the final product of the Build a Blog tut
     const allPosts = await Astro.glob("../pages/posts/*.md");
     const allPosts = await getCollection("posts");
     ---
+    ```
```
Comment (at-the-vr) [inline src/content/docs/en/tutorials/add-content-collections.mdx:251]: I am confused to see how quote block functions just well without closing itself

File: src/content/docs/en/tutorials/add-content-collections.mdx [hunk 2]
```diff
@@ -382,4 +383,4 @@ The steps below show you how to extend the final product of the Build a Blog tut
     }
     ```
 
-For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
\ No newline at end of file
+For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
```
Comment (at-the-vr) [inline src/content/docs/en/tutorials/add-content-collections.mdx:386]: This doesn't makes a visual difference in Deployed site but if its a Astro Docs rule for keeping 2-3 newlines after ending a code block, then I have learned something new

File: src/content/docs/en/tutorials/add-content-collections.mdx (hunk 1 above)
Comment (sarah11918) [reply src/content/docs/en/tutorials/add-content-collections.mdx:251]: I am very confused too!  :sweat_smile:

File: src/content/docs/en/tutorials/add-content-collections.mdx (hunk 2 above)
Comment (sarah11918) [reply src/content/docs/en/tutorials/add-content-collections.mdx:386]: Ah, it's not a convention no (not when it doesn't affect the display) but I noticed the branch I'm currently working on  does have 2 lines, so I wonder whether this was more a copy/paste/revert thing.

We can make it 1 single blank line!

File: src/content/docs/en/tutorials/add-content-collections.mdx [hunk 3]
```diff
@@ -382,4 +383,4 @@ The steps below show you how to extend the final product of the Build a Blog tut
     }
     ```
```
Comment (sarah11918) [inline src/content/docs/en/tutorials/add-content-collections.mdx:385]:
```suggestion
```
Should work fine with just one line here!

File: src/content/docs/en/tutorials/add-content-collections.mdx [hunk 4]
```diff
@@ -381,5 +382,4 @@ The steps below show you how to extend the final product of the Build a Blog tut
       })
     }
     ```
-
-For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
\ No newline at end of file
+For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
```
Comment (sarah11918) [inline src/content/docs/en/tutorials/add-content-collections.mdx:385]:
```suggestion

For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
``` 
Reverting! not sure what happened here!

**Review by sarah11918 — APPROVED** [review]
Thanks @dreyfus92 !  :raised_hands:

**Pull request merged**